bundles/remote_services/discovery_zeroconf/gtest/src/DiscoveryZeroconfAnnouncerTestSuite.cc

```diff
@@ -53,11 +53,11 @@ static int GetLoopBackIfIndex(void);
 class DiscoveryZeroconfAnnouncerTestSuite : public ::testing::Test {
 public:
     static void SetUpTestCase() {
-        (void)system(MDNSD);
+        (void)system("sudo " MDNSD);
```

Comment:
:+1: 

If we need to start processes from C more often (e.g. etcd, msnd, etc) maybe we should also think about running integration test with help of something like [behave](https://pypi.org/project/behave/) or [avocado](https://avocado-framework.github.io/). I think it is even possible to bootstrap behave/avocado from cmake/ctest using a `add_test`

Reply:
I'll find some time to optimize the tests in another new PR. 
I have tried to add FIXTURES in cmake yesterday, but due to some synchronization issues in the current unit tests, and I can't resolve them in a short time. so I'm going to start subprocess from C in this PR.